plugins/GSdx/GSRendererHW.cpp

```diff
@@ -502,7 +502,7 @@ void GSRendererHW::Draw()
 	context->ZBUF.ZMSK = zm != 0;
 
 	// A couple of hack to avoid upscaling issue. So far it seems to impacts mostly sprite
-	if ((m_upscale_multiplier > 1) && (m_vt.m_primclass == GS_SPRITE_CLASS)) {
+	if ((m_upscale_multiplier > 1) && (m_upscale_multiplier < 7) && (m_vt.m_primclass == GS_SPRITE_CLASS)) {
```

Comment:
I never tested custom resolution, the purpose of the check was to disable those hack for native resolution. Path is quite hot so better to keep as light as possible.